### ISSUE TITLE
display_name of Workflow Title overwritten by logged User's display_name on LIVE project

### ISSUE THREAD

**Issue by Jim-Salmons** [zooniverse/Panoptes#1340]
I believe this is the correct project to report this issue against. If not, please advise.

While waiting an extremely long time to hear back on our request for beta approval, I've observed an issue which is significant. 

I've seen the use of display_name across classes result in overwriting of the User email setting and during workflow editing the Workflow Title is being regularly overwritten. This overly aggressive automatic field update is annoying but as long as I change the Workflow Title field to the intended name and quickly leave the page before it is rewritten, no problem. Since this workaround let me get on with our project prototype, no problem.

With our workflow ready for a shakedown cruise we set it to LIVE and applied for Beta status. That's been going on 10 days ago with no action on the approval. 

When a test user reported an issue, I poked around our projects to confirm everything was as it should be. In doing this I visited the DISABLED Workflow page in our Project and watched as the display_name field was overwritten by my User display_name. Okay, I thought... probably cosmetic. But to be sure, I did a quick classification and as you see in this screenshot, the ACTUAL workflow name was changed on a live and uneditable workflow!?

![zooniverse_display_name_workflow_issue](https://cloud.githubusercontent.com/assets/546022/9721370/eef537c6-555e-11e5-80bd-bd582780a131.png)

I can do any requested investigation needed. I am not yet running a local development instance as I am on Windows and don't see any clear instructions on how to set up a Windows-based dev box.

-: Jim :-

**Comment by camallen** [comment]
Hi @Jim-Salmons, the best repo is probably https://github.com/zooniverse/Panoptes-Front-End as this repo is the API backing the front end client. 

I can't reproduce the autosave problem you are experiencing. I'm on Chrome OS X and Linux and the update only fires if there are changes and i change the focus away from the text box. I wonder if this is a bug on windows. Can you please report the OS and browser combo you are experiencing the problem under? Pinging @brian-c for thoughts on this.

Also @mrniaboc, @aliburchard any timeline on the factminer beta review?

**Comment by Jim-Salmons** [comment]
@camallen, thanks for the reply. Will leave this issue here for the moment. You core folks can decide if it needs to move to or be referenced from the front-end repo. (The repo descriptions are very minimal BTW, so a bit of a comment in the respective readme files might make the distinction between between front- and back-end repos more clear for issue reporting.)

I am on a fully updated Win 8.1. And I observed the display_name overwriting in both up-to-date Chrome and Firefox. While the overwriting on a supposed disabled webform was an eye-opener, this behavior was driving me nuts during initial workflow authoring.

@mrniaboc and @aliburchard, I have canceled the FactMiners beta for the moment to nail down some interaction issues and streamline the instructions with a "before you begin" video piece. I'll let you know directly when I resubmit for the beta.

**Comment by brian-c** [comment]
I bet the title field is being overwritten by an autofill plugin. @Jim-Salmons it looks like your LastPass plugin has identified a few matches on zooniverse.org. Could one of them have been associated with the wrong field somehow? Mind turning it off for a moment to see if the problem persists?

**Comment by Jim-Salmons** [comment]
@brian-c It appears that LastPass was the issue! :+1: I just made the tweak and it appears to be settled down. Will update if this is not the case, but here's what I did.

In LastPass I located my Zooniverse logons -- my project-wise admin account, FactMiners, and my personal one -- and went to the Advanced Settings (default hidden) for each logon and checked the "Disable AutoFill" checkbox and saved the entry.

Since the logged-on User account field uses the display_name field which is the same fieldname as a variety of Zooniverse objects, LastPass sees log-in fields where they are not intended.

Considering how many users might have password managers running, it might be reasonable to _store_ the User account's display_name with that database column name, but consider using 'screenname' or 'nickname' in the upper-right logged-on User display and log-on widget-ids. Something that would reduce the chance of to these overwrites happening. If this 'gotcha avoidance' approach is not doable, perhaps a FAQ entry or Wiki comment would be useful. I can write something up w/ screenshots to submit if there is a proper place for such Project Builder docs. (Sorry, I'm new to the community and there are LOTS of repos here, so I don't know the ropes yet.)

Here's a screenshot to clarify...

![image](https://cloud.githubusercontent.com/assets/546022/9742629/0fa77db2-5627-11e5-9fe5-bd78239674e5.png)
